src/vm/isa.rs

```diff
@@ -52,6 +52,22 @@ impl InstructionSet for RgbIsa {
         bset! {"RGB"}
     }
 
+    fn src_regs(&self) -> HashSet<Reg> {
```

Comment:
Can you add some docs to this file? It's not obvious to me what the does.

Reply:
Sure. It returns set of registers whose values are used as inputs. It is very helpful both for printing execution logs (showing inputs and outputs for each opcode) - but also needed for the future static analyzers and code optimizers (if two instructions share no inputs/outputs they are commutative, meaning they can go in any order/parallelized in execution).

It comes from the trait in `AluVM` where it has doc comment, so it should be seen here in docs.rs

Reply:
Cool, I'll look for it there when it's published